src/net/data_stream.rs

```diff
@@ -44,10 +44,10 @@ const HEADER_SIZE: usize = 8;
 // The number of frames that can be buffered.
 const FRAME_COUNT: usize = 4;
 
-// The size of each livestream frame in bytes.
+// The size of each frame in bytes.
 // Ensure the resulting ethernet frame is within the MTU:
 // 1500 MTU - 40 IP6 header - 20 UDP header
-const FRAME_SIZE: usize = 1500 - 40 - 20 - HEADER_SIZE;
+const FRAME_SIZE: usize = 1500 - 40 - 20;
```

Comment:
Note: this was to conservative before and didn't get adapted in some previous refactoring.